pygmt/helpers/decorators.py

```diff
@@ -163,6 +163,9 @@
             (:gmt-term:`PROJ_MEAN_RADIUS`), and the specification of latitude type
             (:gmt-term:`PROJ_AUX_LATITUDE`). Geodesic distance calculations is also
             controlled by method (:gmt-term:`PROJ_GEODESIC`).""",
+    "l": r"""
+        label : str
+            Add a legend entry for the symbol or line being plotted.""",
```

Comment:
Just checking, do we want to add the complicated `-l` stuff at https://github.com/GenericMappingTools/gmt/blob/6.2.0/doc/rst/source/explain_-l.rst_? And/or link to https://docs.generic-mapping-tools.org/6.2/gmt.html#l-full?

```suggestion
        label : str
            [*label*]\ [**+D**\ *pen*][**+G**\ *gap*][**+H**\ *header*][**+L**\ [*code*/]\ *txt*][**+N**\ *cols*][**+S**\ *size*\ [/*height*]][**+V**\ [*pen*]][**+f**\ *font*][**+g**\ *fill*][**+j**\ *just*][**+o**\ *off*][**+p**\ *pen*][**+s**\ *scale*][**+w**\ *width*]
            Add a legend entry for the symbol or line being plotted.""",
```

Reply:
I could add a link to the GMT docs. I chose not to add the complicated stuff because I thought the argument format could change in the near future with https://github.com/GenericMappingTools/pygmt/issues/1082.

Reply:
Ok, let's just add the link then. I also noticed that `contour` and `grdcontour` could benefit from a link to https://docs.generic-mapping-tools.org/6.2/gmt.html#l-full, but we can leave that for someone else to do in the ESWN workshop :)